src/app/listener.ts

```diff
@@ -42,6 +44,20 @@ export class Listener {
 
       this.container.loggerService.info(`Loaded ${this.container.jobService.size()} jobs...`);
 
+      const slashCommandUploads = Array.from(slashCommands.entries()).map(([key, command]) => {
```

Comment:
On second thought this is better off being right after loading from the file system, [here](https://github.com/rob-3/lion-clean-room/blob/360f1c35f24ea8b3690f12cdb9a5af94a5328eff/src/app/bot.ts#L77-L99)

This doesn't really have anything to do with the listener per se and would fix the race condition problem

Reply:
Ok, I will move it.